main.py

```diff
@@ -0,0 +1,32 @@
+from app.animal import Animal, Carnivore, Herbivore
+
+
+if __name__ == "__main__":
+    lion = Carnivore("Simba")
+    print(len(Animal.alive) == 1)
+    print(isinstance(Animal.alive[0], Carnivore) is True)
```

Comment:
Code Efficiency: You can use the `assert` statement here as well, like `assert isinstance(Animal.alive[0], Carnivore)`.